SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -245,55 +77,78 @@ if [ $1 -eq 1 ]; then # Package install
 fi
```

Comment:
Just checking: we are not enabling tardev-snapshotter.service in the new UVM build steps, right?